src/libs/actions/IOU.js

```diff
@@ -2417,16 +2417,6 @@ function getPayMoneyRequestParams(chatReport, iouReport, recipient, paymentMetho
                 lastMessageHtml: optimisticIOUReportAction.message[0].html,
             },
         },
-        {
-            onyxMethod: Onyx.METHOD.MERGE,
-            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${iouReport.reportID}`,
-            value: {
-                [optimisticIOUReportAction.reportActionID]: {
-                    ...optimisticIOUReportAction,
-                    pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
-                },
-            },
```

Comment:
Not sure if I understand this change, we do add this action in backend. Are we moving away from Pattern B? I dont see any discussion linked